src/stylesheets/app.scss

```diff
@@ -1,5 +1,5 @@
 @import "../../node_modules/bootstrap/dist/css/bootstrap.css";
-@import "../../node_modules/opds-web-client/lib/stylesheets/app.scss";
+@import "../../node_modules/library-simplified-reusable-components/lib/stylesheets/app.scss";
```

Comment:
Testing this does not work. It doesn't work because that's not the name of the file.
![Screen Shot 2019-03-26 at 5 17 29 PM](https://user-images.githubusercontent.com/1280564/55034553-0cf88300-4fec-11e9-9b6e-27a820bb47e7.png)

The name of the compiled css is `reusable-components.css` (https://github.com/NYPL-Simplified/reusable-components/blob/master/webpack.prod.config.js#L25) and it should be retrieved from the `dist` folder since that's the webpack compiled folder.

Also, a better way to import scss or css in a project with webpack is to use `~` since webpack understands to use the `node_modules` folder and you can remove the relative path:
```
@import "~library-simplified-reusable-components/dist/reusable-components.css";
```
(I had wanted to but never got a chance to update all the previous imports).

Reply:
Ah.  That explains it.  Thank you; good catch!